pkg/config/config.go

```diff
@@ -192,8 +192,9 @@ func init() {
 
 	// Go_expvar server port
 	Datadog.SetDefault("expvar_port", "5000")
-	// Process Agent
-	BindEnvAndSetDefault("process_agent_enabled", true) // this is to support the transition to the new config file
```

Comment:
@DataDog/kenafeh do you still use this `process_agent_enabled` option for the containerized agent? Or can we safely remove it?

Reply:
I think we should leave this enabled by default. I think @DataDog/burrito will know better.

Reply:
None of our code looks up for this envvar. We inconditionnaly run the process-agent unit, and let it decide whether to stay up or down, if it manages to connect to docker. This can be safely removed as their handling of the `DD_PROCESS_AGENT_ENABLED` envvar does not use `pkg/config`, see https://github.com/DataDog/datadog-process-agent/blob/master/config/config.go#L291

Reply:
great, thanks for the explanation, I'll merge this as-is then 👍